index.js

```diff
@@ -40,6 +42,9 @@ function Consumer(options) {
   this.messageAttributeNames = options.messageAttributeNames || [];
   this.stopped = true;
   this.batchSize = options.batchSize || 1;
+  this.visibilityTimeout = options.visibilityTimeout || 0;
```

Comment:
I think the default for this is actually 30 seconds, which is why I'd rather not include a default value here.